uint8 axis
uint8 requestedControlMode
---
bool success
